common/source/docs/common-telemetry-landingpage.rst

```diff
@@ -19,6 +19,7 @@ information based upon your set-up.
     Bluetooth <common-mission-planner-bluetooth-connectivity>
     ESP8266 wifi telemetry <common-esp8266-telemetry>
     FlightDeck FrSky Transmitter App <common-frsky-flightdeck>
+    Yaapu FrSky OpenTX LUA Script <common-frsky-yaapu>
```

Comment:
same here "Yaapu FrSky Telemetry Script for OpenTX"

Reply:
no problem , will do

Reply:
@Hwurzburg ok thanks